server.js

```diff
@@ -12,6 +12,16 @@ if (process.env.TEST_ENV) {
     return;
 }
 
+console.log('Node Version:', process.version);
+var version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
+if (Number(version[1] <= 4)) {
+  // 5 seems to be where the majority of ES6 was added without flags.
+  // Seems safeist to not allow api to run
+  throw new Error('Node version ' + process.version + ' is not supported');
+}
+if (Number(version[1]) < 6 && Number(version[2]) < 10) {
+  console.error('This node version may not be supported');
```

Comment:
Also check with @browndav but I think we're shipping 6.9 on medic-os so probably best to use that.